src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementResourceApiTest_moveToCell.java

```diff
@@ -0,0 +1,192 @@
+package net.syscon.elite.api.resource.impl;
+
+import net.syscon.elite.api.model.OffenderBooking;
+import net.syscon.elite.service.MovementUpdateService;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpMethod.PUT;
+import static org.springframework.http.HttpStatus.*;
+
+// TODO DT-235 These tests are here to help define the API, but currently using canned data.  Once MovementUpdateService is implemented, replace below with real data.
+public class OffenderMovementResourceApiTest_moveToCell extends ResourceTest {
```

Comment:
I tried to implement this as an @WebMvcTest but that involved major surgery on the configuration to isolate the web components and I deemed it too big a change for a test. :(